### DISCUSSION TITLE
Add support for Github Codespaces

### DISCUSSION THREAD

**Discussion by RyanTheAvian** [KoboldAI/KoboldAI-Client#279] [Ideas]
As the title suggests, I would like to see a version of the Colab made for Github's own Codespaces. I'd love to see a return on Pygmalion as well to the notebook, since Google really hates them. The current colab notebook partially works on Codespaces, but it gives out this error: 
![image](https://user-images.githubusercontent.com/41594800/229961967-7031b2cf-63eb-4779-abae-add5c9524643.png)

**Comment by henk717** [comment]
I do not plan to support python ipynb notebooks outside of colab because they lack features to do right. But it looks like that has terminal access right? If you try to run play.sh --remote it will probably work.
